SCClassLibrary/Common/Geometry/Rect.sc

```diff
@@ -49,9 +49,13 @@ Rect {
 	right_ { |r| left = left - (this.right - r) }
 
 	leftTop { ^Point.new(this.left, this.top) }
```

Comment:
Sure, I see your point, and I was going to ask about this... 
I was refactoring some of the plotter convenience methods into more generally useful ones, and was going to try dispatching some methods with keys I'd already implemented as `topLeft`, etc. But I agree it's not worth muddling the methods in Rect further.

Reply:
More specifically, I've added `Rect:-anchorTo`.

https://github.com/mtmccrea/supercollider/blob/30d062651c4ca4f133e03e8992dd115e2fa3f440/SCClassLibrary/Common/Geometry/Rect.sc#L72

```supercollider
anchorTo { arg aPoint, position;
		var p = aPoint;
		^switch(position,
			// center_ returns new Rect, side setters modify this
			\center,      { this.center_(p) },
			\top,         { this.center_(p).top_(p.y) },
			\bottom,      { this.center_(p).bottom_(p.y) },
			\left,        { this.center_(p).left_(p.x) },
			\right,       { this.center_(p).right_(p.x) },
			\topLeft,     { this.copy.top_(p.y).left_(p.x) },
			\topRight,    { this.copy.top_(p.y).right_(p.x) },
			\bottomLeft,  { this.copy.bottom_(p.y).left_(p.x) },
			\bottomRight, { this.copy.bottom_(p.y).right_(p.x) }
		)
	}
```
Would you prefer that these keys match similar (getter) methods in `Rect`? E.g. `\bottomRight` would be changed to `\rightBottom`. This would then mean the keys change in `DrawGridX:-labelAnchor_`:

https://github.com/mtmccrea/supercollider/blob/30d062651c4ca4f133e03e8992dd115e2fa3f440/SCClassLibrary/Common/GUI/Base/Grid.sc#L225

```supercollider
labelAnchor_ { |anchor|
		var anchorSym = anchor.asSymbol;
		var anchorList = #[
			\center, \top, \bottom, \left, \right, \topLeft, \topRight, \bottomLeft, \bottomRight
		];
...
```

Reply:
yes, I think that `anchorTo` is a good idea.
 I'd keep the whole convention uniform and match the methods in `Rect`.